MVP/defaults-build.yaml

```diff
@@ -57,30 +59,56 @@
                     echo "TESTS=$TESTS" >> $WORKSPACE/job.properties
                 - inject:
                     properties-file: $WORKSPACE/job.properties
+        - postbuildscript:
+            script-only-if-succeeded: False
+            builders:
+                - shell: |
+                    # Steps to generate default/custom test result message to CI
+
+                    # Clone kernel-ci repo somewhere
+                    # Source libcim.sh
+                    cd kernel-ci
+                    export PROJECT_BUILDER_PACKROOT=$WORKSPACE/shared
+                    env >> $WORKSPACE/job.properties
+                    source $PROJECT_BUILDER_PACKROOT/lib/bash/libcim.sh
+
+                    # Call cim_puten if you want to customize some values in the message
+                    # Otherwise cim_injectenv will generate message by best guess
+                    cim_putenv "body.artifact.component" "kdump"
+                    cim_putenv "body.namespace" "kernel-qe.kernel-ci.your_team.kdump"
+                    cim_putenv "body.type" "tier0"
```

Comment:
Should these examples be commented out by default since they are optional?  This would also avoid users accidentally leaving their component as 'kdump' or namespace including 'your_team'.